ext/lib/crypto/mbedtls/Kconfig.tls-generic

```diff
@@ -135,16 +181,45 @@ config TLS_CIPHER_DES_ENABLED
 	bool "Enable the DES block cipher"
 	default y
 
+config TLS_CIPHER_ARC4_ENABLED
+	bool "Enable the ARCFOUR stream cipher"
```

Comment:
Well, it's still "ARC4", I'd say. Actually, it's RC4 in normal speak even, where A in "ARC4" is for "alleged".